lib/store/dashboard/transaction_filter_store.dart

```diff
@@ -24,10 +23,40 @@ abstract class TransactionFilterStoreBase with Store {
   DateTime? endDate;
 
   @action
-  void toggleIncoming() => displayIncoming = !displayIncoming;
+  void toggleIAll() {
+    displayAll.value = (!displayAll.value);
+    if (displayAll.value) {
+      displayOutgoing.value = true;
+      displayIncoming.value = true;
+    }
+    if (!displayAll.value) {
+      displayOutgoing.value = false;
+      displayIncoming.value = false;
+    }
```

Comment:
assign with `displayAll` instead of if

Reply:
@OmarHatem28 I think we should still use if, because computed `displayAll` can't reverse itself, other comments fixed.